File: mobile/src/main/java/com/ourygo/oy/util/DebugUtil.java
package com.ourygo.oy.util;

import android.util.Log;

public class DebugUtil {

    private static long lastTime=0;

    public static void time(String tag){
        long currentTime=System.currentTimeMillis();
        Log.e(tag,currentTime-lastTime+"");
        lastTime=currentTime;
    }

}

File: mobile/src/main/java/com/ourygo/ygomobile/util/DebugUtil.kt
package com.ourygo.ygomobile.util

import android.util.Log

object DebugUtil {
    private var lastTime: Long = 0
    fun time(tag: String?) {
        val currentTime = System.currentTimeMillis()
        Log.e(tag, (currentTime - lastTime).toString() + "")
        lastTime = currentTime
    }
}